packages/extractor/src/api/extractor/extract.ts

```diff
@@ -510,13 +528,13 @@ function sigDocsFromType(docs: Doc.Manager, t: tsm.Type): Doc.DocSig[] {
  * Extract docs from the type's properties.
  */
 function propertyDocsFromType(docs: Doc.Manager, t: tsm.Type): Doc.DocProp[] {
-  return getProperties(t).map((p) => {
-    const propName = p.getName()
-    const propType = p.getType()
+  return getProperties(t).filter(Boolean).map((p) => {
```

Comment:
Hm, interesting! I don't think `getProperties` should allow null/undefined to get through, so probably the fix is there.